Winch/Winch.csproj

```diff
@@ -17,8 +17,9 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="DredgeGameLibs" Version="1.0.4.1" />
 		<PackageReference Include="Lib.Harmony" Version="2.2.2" />
-		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Any particular reason for kicking up the version in this pr?

Reply:
I'm not sure I like how the phrasing of that comment sounds. Let me rephrase:
This increase in version seems accidental/incidental. Is there are reason for it for this pr? I don't think it needs to be removed, just curious

Reply:
I specifically chose that version (13.0.1) as it's the one used by DREDGE, didn't know if it made a huge difference but just wanted to make sure.

Reply:
Ah I see, thanks for clarifying!

Reply:
Not any reasoning to it, feel free to change it back to 13.0.1